src/adapters/sonobi.js

```diff
@@ -0,0 +1,87 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var utils = require('../utils');
+
+var SonobiAdapter = function SonobiAdapter(){
+  var trinity_ep = 'http://apex.go.sonobi.com/trinity.js?key_maker=';
```

Comment:
Do you have a `HTTPS` endpoint? We are requiring all adapters to use secure tags now.